mycpp/gc_mylib.h

```diff
@@ -132,16 +132,13 @@ class Writer : public Obj {
   virtual bool isatty() = 0;
 };
 
-class Buf;  // forward declaration
-
-Str* StrFromBuf(const Buf&);
-Buf* NewBuf(int);
+class MutableStr;
 
 constexpr uint16_t maskof_BufWriter();
 
 class BufWriter : public Writer {
  public:
-  BufWriter() : Writer(Tag::FixedSize, maskof_BufWriter(), sizeof(BufWriter)) {
+  BufWriter() : Writer(Tag::FixedSize, maskof_BufWriter(), sizeof(BufWriter)), str_(0), len_(0) {
```

Comment:
for pointer types, please spell the constant `nullptr` instead of `0`

Reply:
Done

Reply:
Done